webapp/src/components/progress_bars/InformationBar.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="mb-4 space-y-4">
-    <div class="flex flex-wrap text-3xl text-slate-600 justify-center">
```

Comment:
I think the dark theme value isn't used here because <DISCO/> uses its own font color. same for `text-slate-600`